civicrm_entity.routing.yml

```diff
@@ -5,3 +5,11 @@ civicrm_entity.admin:
     _title: 'CiviCRM Entity'
   requirements:
     _permission: 'access administration pages'
+
+civicrm_entity.settings:
+  path: '/admin/structure/civicrm-entity/settings'
+  defaults:
+    _form: '\Drupal\civicrm_entity\Form\CivicrmEntitySettings'
+    _title: 'Settings'
+  requirements:
+    _permission: 'access administration pages'
```

Comment:
This should have a more specific permission - probably a custom settings permission from this module. We usually give site managers "access administration pages" just to get `/admin` and then more specific permissions for other stuff

Reply:
in the D7 version we have 'administer CiviCRM Entity'